docs/docs/1.2.3/policies/traffic-metrics.md

```diff
@@ -136,71 +136,75 @@ metrics:
     path: /non-standard-path
 ```
 
-As a result, this particular dataplane will expose an HTTP endpoint with `Prometheus` metrics on port `1234` and URI path `/non-standard-path`.
+This proxy exposes an HTTP endpoint with Prometheus metrics on port `1234` and URI path `/non-standard-path`.
 :::
 ::::
 
 ### Configure Prometheus
 
-Although dataplane metrics are now exposed, `Prometheus` doesn't know anything about it just yet.
+Although proxy metrics are now exposed, you still need to let Prometheus discover them.
 
-To help `Prometheus` to automatically discover dataplanes, `Kuma` provides a tool - `kuma-prometheus-sd`.
-`kuma-prometheus-sd` is meant to run alongside `Prometheus` instance.
-It knows location of `Kuma` Control Plane is and can fetch an up-to-date list of dataplanes from it.
-It then transforms that information into a format that `Prometheus` can understand, and saves it into a file on disk.
-`Prometheus` watches for changes to that file and updates its scraping configuration accordingly.
+In Prometheus version 2.29 and later, you can add Kuma metrics to your `prometheus.yml`:
+
+```sh
+kuma_sd_configs:
+ - server: http://kuma-control-plane.kuma-system.svc:5676
+```
+
+For earlier versions of Prometheus, Kuma provides the `kuma-prometheus-sd` tool, which runs alongside your Prometheus instance.
+This tool fetches a list of current data plane proxies from the Kuma control plane and saves the list in Prometheus-compatible format 
+to a file on disk. Prometheus wathces for changes to the file and updates its scraping configuration accordingly.
 
 :::: tabs :options="{ useUrlFragment: false }"
 ::: tab "Kubernetes"
-Use `kumactl install metrics | kubectl apply -f -` to deploy configured Prometheus with Grafana.
 
-::: tip
-If you've got Prometheus deployment already, you can use [Prometheus federation](https://prometheus.io/docs/prometheus/latest/federation/) to bring Kuma metrics to your main Prometheus cluster.
+You can run `kumactl install metrics | kubectl apply -f -` to deploy configured Prometheus with Grafana.
```

Comment:
Feels like this line should be at the top. It's the simplest most straight forward way to do this.

Then you can add a section about "add a configuration to scrape Kuma dataplane proxies" from your own prometheus and then do version > 2.29 or the other.

People that just want to get started should just run `install metrics` and not have to care how prometheus configuration works.

Reply:
It's at the top of the K8s section already. Any further up and it would look like you can run kubectl on Universal ;)